ras_party/controllers/party_controller.py

```diff
@@ -623,17 +622,12 @@ def resend_verification_email(party_uuid):
     :param party_uuid: the party uuid
     :return: make_response
     """
-    log.debug('attempting to resend verification_email with party_uuid {}'.format(party_uuid))
+    log.debug('Attempting to resend verification_email with party_uuid {}'.format(party_uuid))
```

Comment:
Does this need to be updated to conform with the logging standards?

Reply:
It does but all the other logging in party service needs to be updated to meet those standards as well I think. Maybe we should add to the technical debt?